NewHorizons/Utility/DebugUtilities/DebugPropPlacer.cs

```diff
@@ -39,14 +40,36 @@ private struct PropPlacementData
         public GameObject mostRecentlyPlacedPropGO { get { return props.Count() <= 0 ? null : props[props.Count() - 1].gameObject; } }
         public string mostRecentlyPlacedPropPath { get { return props.Count() <= 0 ? "" : props[props.Count() - 1].detailInfo.path; } }
 
+        private ScreenPrompt _placePrompt;
+        private ScreenPrompt _undoPrompt;
+        private ScreenPrompt _redoPrompt;
+
         private void Awake()
         {
             _rc = this.GetRequiredComponent<DebugRaycaster>();
             currentObject = DEFAULT_OBJECT;
+
+            _placePrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_PLACE", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.G));
+            _undoPrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_UNDO", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.Minus));
+            _redoPrompt = new ScreenPrompt(TranslationHandler.GetTranslation("DEBUG_REDO", TranslationHandler.TextType.UI) + " <CMD>", ImageUtilities.GetButtonSprite(KeyCode.Equals));
```

Comment:
This button is blank

Reply:
Yeah, I know. I was trying to get that bug fixed in Vanilla Fix. https://github.com/JohnCorby/ow-vanilla-fix/pull/5